### DISCUSSION TITLE
Youtube ad blocker detection

### DISCUSSION THREAD

**Discussion by TheFurryGamer2016** [dhowe/AdNauseam#2433] [Q&A]
So I've suddenly started getting the dreaded "Ad blocker detected" popup on a specific video (https://www.youtube.com/watch?v=FVR8zz8ci2k) And was wondering if youtube changed its detection method or if there's some way to fix this. When I disable the application, the video (And ad) play fine, but with it enabled, it doesn't play. (Even without the hard-coded strict block disabled)

This suddenly started happening about an hour ago and I have no idea how to stop it, any possible fixes? (Already disabled the quick fixes tab after strict block and the issue persisted)

(Edit, all changes have been reverted, issue still persists)

**Comment by TheFurryGamer2016** [comment]
I found a fix which was strictly blocking ads with brave, I'll see how long this works, but it might stop in the verry near future

**Comment by mneunomne** [comment]
@TheFurryGamer2016 in the last weeks youtube has been tackling ad-blockers really hard. Since we are based on uBlock Origin and the open-source ad-blocking community, we are overall dependant on them to make improvements in this aspect. Nice that you found a temporary solution, we are working in order to keep as up-to-date as possible with uBlock Origin, and hopefully this issue will gradually decrease as more people work on it.

**Comment by candre23** [comment]
Adnauseam was sneaking under youtube's anti-adblocker radar until today.  Now I'm unable to use youtube without disabling adnauseam.  I guess it's learning.

**Reply by sarisotter** [reply]
Same, now I'm having this problem. If someone can help, it will be appreciated.

**Reply by KillerTheFlareon** [reply]
I'm also having this problem.
You could get around it for a while in incognito for some reason, but that too also stopped working.

**Comment by dhowe** [comment]
See https://github.com/dhowe/AdNauseam/issues/2422
